src/Amadevus.RecordGenerator.Generators/Names.cs

```diff
@@ -1,5 +1,8 @@
 ﻿namespace Amadevus.RecordGenerator.Generators
 {
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Move usings outside namespace here as well

Reply:
But also, remove them as unused, actually. :D

Reply:
> But also, remove them as unused, actually. :D

Done with 2eb52200c0d0f2b1a49258092d3a614d1a0d14d5 though there are lots of those cases all over the project. 😉